ccd-definition/AuthorisationCaseField.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "CaseFieldID": "caseHistory",
+    "UserRole": "caseworker-cmc-solicitor",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "PERSONAL_INJURY",
+    "CaseFieldID": "eligibilityQuestions",
+    "UserRole": "caseworker-cmc-solicitor",
+    "CRUD": "CRU"
```

Comment:
do we need all `CU` if it's only label?

Reply:
Can remove U, without C the label does not appear